fastlane/lib/fastlane/actions/docs/frame_screenshots.md

```diff
@@ -21,19 +21,30 @@ _frameit_ allows you to put a gorgeous device frame around your iOS and macOS sc
 
 # Features
 
+## Frame screenshot
+
 Put a gorgeous device frame around your iOS and macOS screenshots just by running one simple command. Support for:
 
 - iPhone, iPad and Mac
 - Portrait and Landscape modes
-- Several colors
+- Several device frame colors
 
 The complete and updated list of supported devices and colors can be found [here](https://github.com/fastlane/frameit-frames/tree/gh-pages/latest)
 
 Here is a nice gif, that shows _frameit_ in action:
 
 ![img/actions/FrameitGit.gif](/img/actions/FrameitGit.gif?raw=1)
 
-### Results
+## Advanced Features
+
+- Put framed screenshot on colored background, define padding
+- add text above or under framed screenshot
+  - keyword + text
+  - choose text font and color
+  - multi line text
+  - "intelligent" positioning of text that always looks good(ish)
```

Comment:
Personally, I'd change these to:

```
- Add text above or under the framed screenshot, allowing you to:
  - Add a keyword along with the text (not sure about this)
  - Choose the font and color
  - Make it multilined
  - Automatic positioning for almost perfect looks
```

Reply:
> Add a keyword along with the text (not sure about this)

Nah, "keyword" and "text" are referring to the "text fields" being in use here. Pretty technical (and bad) terms but defined in code.

Reply:
What about the rest of the changes? Including uppercased items.

Reply:
I don't care - but sounds good.

Reply:
Some changes that better reflect the frameit terms:

- Add text above or under the framed screenshot, allowing you to:
  - This can be a keyword with optional title
  - Choose the font and color
  - frameit supports multi-line text
  - Automatic positioning for almost perfect looks